tests/runner/src/main/java/org/mule/test/runner/utils/RunnerModuleUtils.java

```diff
@@ -155,8 +159,9 @@ private static void addSdkApiClassesDynamically(ClassLoader extensionClassLoader
           ClassLoadingStrategy.Default.INJECTION.load(extensionClassLoader, mapEntry);
           addedClasses.add(classEntry.getKey().getName());
           classAdded = true;
+          LOGGER.debug("Class {} was succesfully added to the extension classloader.", classEntry.getKey().getName());
         } catch (Exception e) {
-          // Problem found adding class to class loader.
+          LOGGER.debug("Class {} failed to be added to the extension classloader.", classEntry.getKey().getName());
```

Comment:
also log the exception and its message

Reply:
is there a way to separate the wrong class order case from other errors which shouldn't be ignored?